docs/managing-compute-resources.asciidoc

```diff
@@ -64,13 +64,13 @@ spec:
             cpu: 2
 ----
 
-<1> Replace with `kibana` or `apm-server` as appropriate.
+For `containers` name, you can use `apm-server` for `kibana` as appropriate.
```

Comment:
```suggestion
For the container name, you can use `apm-server` for `kibana` as appropriate.
```

Since we are just describing one container in a list of containers